pkg/templates/templates_test.go

```diff
@@ -66,5 +70,17 @@ func TestParseTruncateFunction(t *testing.T) {
 		var b bytes.Buffer
 		assert.NoError(t, tm.Execute(&b, source))
 		assert.Equal(t, testCase.expected, b.String())
+
+		t.Run("Empty Source Test with template: "+testCase.template, func(t *testing.T) {
```

Comment:
Can we either convert the test above this to use the `t.Run` pattern or continue to use the already established pattern in this test?